airflow/bin/cli.py

```diff
@@ -641,6 +641,11 @@ def get_parser():
 
     ht = "Burn down and rebuild the metadata database"
     parser_resetdb = subparsers.add_parser('resetdb', help=ht)
+    parser_resetdb.add_argument(
```

Comment:
Should we `help=argparse.SUPPRESS`?

I was almost thinking about removing `resetdb` altogether as we are growing our user base beyond control at Airbnb.